proxy/core/src/main/scala/io/cloudstate/proxy/crdt/CrdtSupportFactory.scala

```diff
@@ -93,9 +94,10 @@ class CrdtSupportFactory(system: ActorSystem,
     val methodsWithoutKeys = methodDescriptors.values.filter(_.keyFieldsCount < 1)
     if (methodsWithoutKeys.nonEmpty) {
       val offendingMethods = methodsWithoutKeys.map(_.method.getName).mkString(",")
-      throw new EntityDiscoveryException(
-        s"CRDT entities do not support methods whose parameters do not have at least one field marked as entity_key, " +
-        "but ${serviceDescriptor.getFullName} has the following methods without keys: ${offendingMethods}"
+      throw EntityDiscoveryException(
+        s"""CRDT entities do not support methods whose parameters do not have at least one field marked as entity_key,
+            |but ${serviceDescriptor.getFullName} has the following methods without keys: ${offendingMethods}""".stripMargin
+          .replaceAll("\n", " ")
```

Comment:
Is this to fix the string interpolation missing from the second string? Rather than multiline string, with margin stripped, and then newlines replaced, could just add string interpolation to the second string (`s"`) and not the first one.